README.md

```diff
@@ -15,7 +15,6 @@ This service requires the following dependent services:
 
 ## Alerts
 
-- Details of relevant alerts can be obtained from the `hmpps-prodiner-profile` repository [here](https://github.com/ministryofjustice/hmpps-prisoner-profile/blob/main/server/data/alertFlags/alertFlags.ts).
```

Comment:
This link is a 404.
Also prisoner profile just uses the alerts API now so no information available